### PR TITLE
Sql server ao-ag using single subnet

### PR DESCRIPTION
**Pull request by shashank-google**

### CODE REVIEW

**Comment by ToddKopriva** [comment]
Has this received a technical review yet?

**Comment by shashank-google** [comment]
@ToddKopriva this needs a technical review.
I am hunting for a reviewer.

**Comment by shashank-google** [comment]
Tagging @sqlthinker (Anibal) who is technical reviewer.

File: tutorials/sql-server-ao-single-subnet/index.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
+---
+title: Deply Microsoft SQL Server Always On in a single subnet
```
Comment (chbussler) [inline tutorials/sql-server-ao-single-subnet/index.md:2]: Spelling: Deploy, not Deply

Comment (chbussler) [inline tutorials/sql-server-ao-single-subnet/index.md:2]: Uppercase Single Subnet
Maybe Single VPC Subnet?

File: tutorials/sql-server-ao-single-subnet/index.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    --boot-disk-size "200" \
+    --boot-disk-type "pd-ssd" \
+    --boot-disk-device-name "node-2" \
+    --metadata enable-wsfc=true,sysprep-specialize-script-ps1="Install-WindowsFeature Failover-Clustering -IncludeManagementTools;"
+```
+
+## Creating a file share witness
+To provide a tie-breaking vote and achieve a [quorum](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/jj612870(v=ws.11))
+for the failover scenario, create a file share that will act as a witness. For the purposes of this tutorial, you will create the file share witness on the domain controller VM. 
+In a production environment, you would create it elsewhere.
```
Comment (chbussler) [inline tutorials/sql-server-ao-single-subnet/index.md:144]: Where would that be? Could you describe a good production location?

**Comment by chbussler** [comment]
LGTM

**Comment by ToddKopriva** [comment]
Thanks for the review, @chbussler .

I'll probably finish with the editorial and production pass tomorrow..

**Review by ToddKopriva — APPROVED** [review]

**Pull request merged**